package.json

```diff
@@ -53,12 +53,14 @@
     "react-final-form": "6.5.9",
     "react-select": "5.8.0",
     "rehype-external-links": "3.0.0",
+    "rehype-pretty-code": "0.13.2",
     "rehype-stringify": "9.0.3",
     "remark": "14.0.3",
     "remark-html": "15.0.2",
     "remark-parse": "10.0.2",
     "remark-rehype": "10.1.0",
     "secure-password": "4.0.0",
+    "shiki": "1.5.2",
```

Comment:
is this needed? i dont see it used

Reply:
Yes, this is needed dependency.

Reply:
I think highlight.js will be better so we don't need this, but good to know we can bump the others safely! i will merge these into the other branch